app/assets/stylesheets/welcome.scss

```diff
@@ -149,3 +149,11 @@ span.list-group-item.active {
   border: black;
   font-size: 22px;
 }
+
+.font-icon-clicks {
+  color: #e573fc;
```

Comment:
Color literals like `#e573fc` should only be used in variable declarations; they should be referred to via variable everywhere else.